Snyk.Code.Library/Service/BundleService.cs

```
@@ -61,11 +61,22 @@ public async Task<Bundle> CheckBundleAsync(string bundleId, CancellationToken ca
             {
                 var pathToHashAndContentDict = codeCacheService.CreateFilePathToHashAndContentDictionary(resultBundle.MissingFiles);
 
-                await this.UploadFilesAsync(resultBundle.Id, pathToHashAndContentDict, fireScanCodeProgressUpdate, cancellationToken: cancellationToken);
+                try
+                {
+                    await this.UploadFilesAsync(resultBundle.Id, pathToHashAndContentDict, fireScanCodeProgressUpdate, cancellationToken: cancellationToken);
+                }
+                catch (Exception e)
+                {
+                    Logger.Error(e, "Error on upload missing files");
```

We swallow an exception here. Does this mean we rely on retry for exception to go away? 
Can we have more documentation here to understand the use case?